assembl/static2/css/variables.scss

```diff
@@ -86,6 +86,12 @@ $img-dir: '/static2/img';
   src: local('☺'), url('#{$fonts-dir}/Lato-LightItalic.woff') format('woff'), url('#{$fonts-dir}/Lato-LightItalic.ttf') format('truetype');
 }
 
+@font-face {
```

Comment:
If you're going to add this font-family to our css, you should also add this to our styleguide. Nobody outside of those who review this PR will know of this in the future. @paolina92 thoughts?